testing/docker/kibana/kibana.yml

```diff
@@ -19,3 +19,5 @@ xpack.fleet.agentPolicies:
         id: default-fleet-server
         package:
           name: fleet_server
+
+xpack.profiling.enabled: true
```

Comment:
Intentionally added to speed up development: if you prefer to remove it let me know

Reply:
I think this is fine, thanks.